wagn/lib/wagn/all.rb

```diff
@@ -1,2 +1,2 @@
-# backwards compatibility
-require 'wagn/application'
\ No newline at end of file
+require 'decko/engine'
+require 'wagn/application'
```

Comment:
I didn't thinks so.  This is the engine that is now in the wagn gem, but in Decko namespace.

I think we almost always need it because it configures (and probably causes to load) Card.  Are you saying it shouldn't always load with wagn/all.  I change the generator back, so 'compatibility mode' is the new mode too.